scripts/importing-Exporting-Config.js

```diff
@@ -0,0 +1,480 @@
+{
+    var loc_Strge_Keys = Object.keys(localStorage);
+    var total_Output_Obj = {};
+    /*
+        --> Event A
+            --> Team B
+                --> Match data
+                --> Pit Data
+            --> Team C
+            ...
+    */
+   
+    for (var i = 0; i < eventsArr.length; i++) //loop through events in a season
+    {
+        var currEvent = eventsArr[i];
+        //var event_Obj = {};
+        var event_Obj_Name = `${currEvent}`;
+        var event_Obj_Value = {};
+        
+        var teams_From_Cookie = getCookie(event_data_cookie_name(season,currEvent));
+        if (teams_From_Cookie.trim().length == 0) continue;
+        
+        var teams_In_Curr_Event = JSON.parse(teams_From_Cookie); //object having kv pairs of format teamNum:teamName
+        var event_Specific_Teams_Keys = Object.keys(teams_In_Curr_Event); //has team nums
+
+        for (var j = 0; j < event_Specific_Teams_Keys.length; j++) //loop through teams
+        {
+            var currTeam = event_Specific_Teams_Keys[j];
+            var currTeamName = getTeams(season,currEvent)[currTeam];
+            //alert(currTeam);
+            
+            //var team_Obj = {};
+            var team_Obj_Name = `${currTeam}:${currTeamName}`;
+            var team_Obj_Value = {}; //will be of length 2: having a match obj and a pit obj
+            
+            //var match_Obj = {};
+            var match_Obj_Name = `match data`;
+            var match_Obj_Value = []; //length is unknown (dont know how many matches per event the team will attend)
+            
+            //var pit_Obj = {}; 
+            var pit_Obj_Name = `pit data`;
+            var pit_Obj_Value = []; //not an array, only 1 value (only 1 pit entry per event)
+
+            var loc_Strg_Check = `/${season}/${currEvent}/${currTeam}/`; //to check if a key contains the curr event/curr team
+
+            for (var k = 0; k < loc_Strge_Keys.length; k++)
+            {
+                var curr_loc_Strg_key = loc_Strge_Keys[k];
+                if (!curr_loc_Strg_key.startsWith(loc_Strg_Check)) continue; //we know curr key has data on selected team for curr event
+
+                if (curr_loc_Strg_key.includes('Match Data')) //add to match_Obj
+                {
+                    var match_Data_Arr = JSON.parse(localStorage[curr_loc_Strg_key]); //is an array of smaller element/frequency object pairs
+                    match_Obj_Value.push(match_Data_Arr);
+                }
+
+                if (curr_loc_Strg_key.includes('pit')) //add to pit_Obj
+                {
+                    var pit_Data_Arr = JSON.parse(localStorage[curr_loc_Strg_key]); //is an array of smaller Q/A object pairs
+                    pit_Obj_Value = pit_Data_Arr;
+                }
+            }
+
+            //set up match/pit objects
+            //match_Obj[match_Obj_Name] = match_Obj_Value;
+            //pit_Obj[pit_Obj_Name] = pit_Obj_Value;
```

Comment:
don't comment out code. Delete it and make a commit with an explanative message why.